src/components/Catalog/Catalog.tsx

```diff
@@ -1,17 +1,21 @@
+import { useState } from 'react';
 import { ReactComponent as AddIcon } from '../../icons/add.svg';
-import ExistingGraphs from './ExistingGraphs/ExistingGraphs';
+import GraphWorkshop from '../GraphWorkshop/GraphWorkshop';
 import './Catalog.scss';
+import ExistingGraphs from './ExistingGraphs/ExistingGraphs';
 
 const Catalog = () => {
+	const [shouldOpenGraphWorkshop, setShouldOpenGraphWorkshop] = useState(false);
```

Comment:
im not sure what happen to indention in github, if it normal in yours so just resolve this comment